changelogs/7.0.asciidoc

```diff
@@ -17,6 +27,7 @@ https://github.com/elastic/apm-server/compare/v6.7.0\...v7.0.0[View commits]
 [float]
 ==== Breaking Changes
 - Move fields in ES to be ECS compliant {pull}1766[1766],{pull}1783[1783],{pull}1813[1813],{pull}1836[1836],{pull}1838[1838],{pull}1844[1844],{pull}1848[1848],{pull}1849[1849],{pull}1863[1863],{pull}1870[1870]
+// ** [More information link would go here (linking to APM overview breaking changes)]
```

Comment:
I am not sure if it shouldn't be the other way round, to give a highlight in the apm guide, and then link to the exhaustive changes list in the dedicated repositories. 

Apart from that I like the suggested changes.

Reply:
@simitt - I might've done a bad job explaining. I think the exhaustive changes should live in the dedicated repos (where they currently live). 

The biggest issue for me was determining whether the link you highlight here should link to the _APM **Overview** breaking changes_ or the _APM **Server** breaking changes_. Ultimately I went we the **Overview** so that users would also be able to easily see all APM breaking changes in one place. I still added a brief note in the Server breaking changes. 

I've added some more commits to this PR that should make my vision clearer. Take a look and let me know your thoughts.

Reply:
+1 on your proposal and changes. 
One minor note, I think the link from the apm-server to the general APM breaking changes is broken, at least I got a 404 when trying to follow it.

Reply:
Yeah, this is one of the limitations we'll run into when making doc changes in two different books (server & overview) at the same time. Since we can only build one book locally at a time, the new pages in the APM overview will not be built, and the links will 404. I've gone through and manually looked at all the URLs to make sure they look correct, and they do, so I'm hopeful there will be no broken links when this PR is merged 🤞